### PR TITLE
HPCC-28181 update README.md files and handling non default region

### PR DESCRIPTION
**Pull request by xwang2713**
Add help scripts for adding nfs inbound rule and stop EKS cluster

@g-pan please help reviewing following README.md

./EFS-CSI/README.md
./EKS/one-node-pool/README.md
./EKS/README.md
./IAM/README.md
./README.md

### CODE REVIEW

File: EFS-CSI/README.md [hunk 1]
```diff
@@ -14,6 +14,16 @@ EFS_ID           # EFS ID can be found from AWS Console or command-line:
 EFS_REGION       # EFS region
 EFS_SECURITY_GROUPS  # EFS seucurity group which is the subnet security group which should be same as EKS
 ```
+
+## Add NFS Inbound Rule
+Either from AWS Console or run following script to add NFS inbound rule if no such rule available yet.
```
Comment (g-pan) [inline EFS-CSI/README.md]: Either from the AWS Console or run the following script to add the NFS inbound rule if no such rule is present.

File: README.md [hunk 2]
```diff
@@ -56,9 +56,13 @@ NAME            PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      A
 aws-efs         efs.csi.aws.com         Delete          Immediate              false                  29h
 gp2 (default)   kubernetes.io/aws-ebs   Delete          WaitForFirstConsumer   false                  37h
 ```
-cd to helm/ and run
+Currently helm/example/efs is broken due the expired old EFS CSI Provider and there is a PR
```
Comment (g-pan) [inline README.md]: suggestion: Remove the "and there is a PR"

**Review by g-pan — COMMENTED** [review]
minor comments in line.

File: EFS-CSI/README.md (hunk 1 above)
Comment (xwang2713) [reply EFS-CSI/README.md]: fixed

File: README.md (hunk 2 above)
Comment (xwang2713) [reply README.md]: Fixed

**Comment by xwang2713** [comment]
@arjunachala and @dcamper it is ready to merge.

**Pull request merged**